### PR TITLE
migrate benchmark repo

### PR DESCRIPTION
**Pull request by juanpicado**
- Migrate https://github.com/pnpm/benchmarks-of-javascript-package-managers onto this repository
- Enable GitHub action to run Benchmarks
- Commit new benchmarks automatically (disabled for now)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
     "publish-gh-pages": "docusaurus-publish --skip-image-compression",
     "write-translations": "rm -rf i18n/en && docusaurus write-translations",
     "version": "docusaurus docs:version",
+    "prebench:clean": "./node_modules/.bin/rimraf .tmp",
```
Comment (juanpicado) [inline package.json:9]: Not sure if there is a better way with pnpm.

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Benchmark runner
+
+on: push
```
Comment (juanpicado) [inline .github/workflows/benchmark.yml:3]: currently running on every branch, I can limit this to master or schedule to run every day.

Comment (zkochan) [reply .github/workflows/benchmark.yml:3]: I don't think it should run on push. It should run every day.

Comment (juanpicado) [reply .github/workflows/benchmark.yml]: Set on 2:30am every day. Any preference?

**Review by zkochan — APPROVED** [review]

**Comment by zkochan** [comment]
actually, I am not sure about this solution. I thought it would be a subproject with its own package.json. Now its dependencies are mixed with the website deps

**Comment by juanpicado** [comment]
:) ok too late to change it, I can refactor it, maybe you can picture me the idea you have and follow up a PR

**Comment by zkochan** [comment]
I might just move it around myself

**Comment by zkochan** [comment]
moved

**Comment by zkochan** [comment]
ok, I actually amended the change and force pushed now

**Comment by juanpicado** [comment]
Nice touch. `pnpm --dir=benchmarks`, btw `--dir` is not at the docs? I cannot find it.

**Comment by zkochan** [comment]
it is here
https://pnpm.js.org/pnpm-cli

**Pull request merged**